src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -104,9 +117,9 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     <button
       className={containerClasses}
       style={roomInlineStyles}
-      onClick={isCovertRoom ? noop : selectRoomWithSound}
-      onMouseEnter={isCovertRoom ? noop : handleRoomHovered}
-      onMouseLeave={isCovertRoom ? noop : handleRoomUnhovered}
+      onClick={shouldBeClickable ? selectRoomWithSound : noop}
+      onMouseEnter={shouldBeClickable ? handleRoomHovered : noop}
+      onMouseLeave={shouldBeClickable ? handleRoomUnhovered : noop}
```

Comment:
I think by definition these event handlers can accept `undefined` as a value, so no need of `noop` function

Reply:
fixed in 2ab4e9997f9341a3a86c13ea0a77a77b55a15131